bindings/python/test.py

```diff
@@ -165,6 +165,13 @@ def test_post_session_stats(self):
 		self.assertTrue(isinstance(a.values, dict))
 		self.assertTrue(len(a.values) > 0)
 
+	def test_unknown_settings(self):
+		try:
+			s = lt.session({'unexpected-key-name': 42})
+			self.assertFalse('should have thrown an exception')
+		except Exception as e:
```

Comment:
`Exception` is too general, it should match the thrown/raised error `KeyError` (based on change above)